src/types/responses/AccountBalanceInfo.ts

```diff
@@ -7,9 +7,9 @@ export interface IAccountBalanceInfo {
 	at: IAt;
 	tokenSymbol: string;
 	nonce: Index;
-	free: Balance;
-	reserved: Balance;
-	miscFrozen: Balance;
-	feeFrozen: Balance;
+	free: Balance | string;
+	reserved: Balance | string;
+	miscFrozen: Balance | string;
+	feeFrozen: Balance | string;
 	locks: Vec<BalanceLock>;
```

Comment:
We also want to denominate the balance value in each `BalanceLock` and will need a seperate class method to parse the locks and apply the denomination accordingly